woo-razorpay.php

```diff
@@ -1496,7 +1496,7 @@ public function process_refund($orderId, $amount = null, $reason = '')
         }
 
         // process refund for gift card
-        function processGiftCardRefund($orderId, $razorpayPaymentId, $amount = null, $reason = '')
+        function processRefund($orderId, $razorpayPaymentId, $amount = null, $reason = '')
```

Comment:
Already function named with `process_refund` is present. Can we reuse that or give a different name which specify it's related to magic-related for terra-wallet!

Reply:
sure